content/en/about/bugs/index.md

```diff
@@ -66,10 +66,10 @@ containing:
 
 * Current and previous logs from all Istio components and sidecar
 
-* Pilot logs:
+* Istiod logs:
 
     {{< text bash >}}
-    $ kubectl logs -n istio-system -l istio=pilot -c discovery
+    $ kubectl logs -n istio-system -l app=istiod
```

Comment:
Are you sure the `-c` isn't ever needed? Isn't there still the possibility of running istiod with a sidecar?

Reply:
Not today. In a hypothetical future perhaps, but this is probably fine for now. Ed found some annotation to make something the "default" container for logs so we can always add that if we add a sidecar

Reply:
`-c` was required because there used to be a multiple containers for istiod (discovery, istio-proxy) and `istio-proxy` was removed.  

So the following works fine.

```
$ kubectl logs -n istio-system -l app=istiod
2020-09-09T14:15:02.960804Z     info    ads     ADS: "10.244.0.33:32876" router~10.244.0.33~istio-ingressgateway-74b98f8d9c-vllb4.istio-system~istio-system.svc.cluster.local-1 terminated rpc error: code = Canceled desc = context canceled
2020-09-09T14:15:03.389573Z     info    ads     ADS: new connection for node:router~10.244.0.33~istio-ingressgateway-74b98f8d9c-vllb4.istio-system~istio-system.svc.cluster.local-2
2020-09-09T14:15:03.389806Z     info    ads     CDS: PUSH for node:istio-ingressgateway-74b98f8d9c-vllb4.istio-system resources:19
2020-09-09T14:15:03.389854Z     info    ads     EDS: PUSH for node:istio-ingressgateway-74b98f8d9c-vllb4.istio-system resources:18 empty:0 cached:18/18
2020-09-09T14:15:03.389909Z     info    ads     LDS: PUSH for node:istio-ingressgateway-74b98f8d9c-vllb4.istio-system resources:0
2020-09-09T14:43:36.447992Z     info    ads     ADS: "10.244.0.33:34080" router~10.244.0.33~istio-ingressgateway-74b98f8d9c-vllb4.istio-system~istio-system.svc.cluster.local-2 terminated rpc error: code = Canceled desc = context canceled
2020-09-09T14:43:36.926592Z     info    ads     ADS: new connection for node:router~10.244.0.33~istio-ingressgateway-74b98f8d9c-vllb4.istio-system~istio-system.svc.cluster.local-3
2020-09-09T14:43:36.927090Z     info    ads     CDS: PUSH for node:istio-ingressgateway-74b98f8d9c-vllb4.istio-system resources:19
2020-09-09T14:43:36.927202Z     info    ads     EDS: PUSH for node:istio-ingressgateway-74b98f8d9c-vllb4.istio-system resources:18 empty:0 cached:18/18
2020-09-09T14:43:36.927313Z     info    ads     LDS: PUSH for node:istio-ingressgateway-74b98f8d9c-vllb4.istio-system resources:0
```

Reply:
There is only one container for istiod.

```
$ kubectl get pod istiod-6fc8557fbc-kgx2n -n istio-system -o jsonpath="{..image}"
gcr.io/istio-testing/pilot:1.8-dev gcr.io/istio-testing/pilot:1.8-dev
```


`istio-proxy` will be added back but not sure when. https://github.com/istio/istio/pull/24821

Reply:
I assume we can back port this to 1.7 as well (as I changed similar stuff last week).

Reply:
LGTM